src/test/java/org/jenkinsci/plugins/oic/SecurityRealmConfigurationFIPSTest.java

```diff
@@ -3,36 +3,23 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mockStatic;
 
 import hudson.model.Descriptor;
 import jenkins.security.FIPS140;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
 
 class SecurityRealmConfigurationFIPSTest {
 
-    private static Object fipsProperty;
-
-    @BeforeAll
-    static void setUp() {
-        fipsProperty = System.getProperties().setProperty(FIPS140.class.getName() + ".COMPLIANCE", "true");
-    }
-
-    @AfterAll
-    static void tearDown() {
-        if (fipsProperty != null) {
-            System.setProperty(FIPS140.class.getName() + ".COMPLIANCE", String.valueOf(fipsProperty));
-        } else {
-            System.clearProperty(FIPS140.class.getName() + ".COMPLIANCE");
-        }
-    }
```

Comment:
the removal here is no longer testing that the options work correctly whn run in FIPS mode e.g. `escapeHatchToFalse()` is not running in FIPS mode and so the test in here is now invalid.